### PR TITLE
updated azure feed ut

### PR DESCRIPTION
**Pull request by teizenman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24154

## Description
Removed the `get-indicators` test since its in the PB, added mock for download link extraction.

### CODE REVIEW

File: Packs/FeedAzure/Integrations/FeedAzure/FeedAzure_test.py [hunk 1]
```diff
@@ -192,15 +197,3 @@ def test_extract_metadata(regions_list, services_list, values_group_section, exp
...
-def test_get_indicators(mocker):
-    mocker.patch.object(demisto, 'args', return_value={
-        'regions': 'norwaye, westus',
-        'services': 'AzureBackup, AzureAppService',
-    })
-    mocker.patch.object(demisto, 'command', return_value='azure-get-indicators')
-    mocker.patch.object(demisto, 'results')
-
-    main()
-    assert demisto.results.call_args[0][0]['Contents']
```
Comment (ShahafBenYakir) [inline Packs/FeedAzure/Integrations/FeedAzure/FeedAzure_test.py:206]: Why have we deleted this?

File: Packs/FeedAzure/Integrations/FeedAzure/test_data/response_mock.txt [hunk 2]
```diff
@@ -0,0 +1,247 @@
...d&quot;:&quot;nn4c3c1m1r1a1&quot;,&quot;sN&quot;:4,&quot;aN&quot;:&quot;c3c1m1r1a1&quot;}"></button><span data-global-title="Microsoft home" class="js-mobile-title">Download Center</span><button type="button" class="c-action-trigger c-glyph glyph-chevron-right" aria-label="See more menu options" data-m="{&quot;cN&quot;:&quot;Mobile forward button_nonnav&quot;,&quot;id&quot;:&quot;nn5c3c1m1r1a1&quot;,&quot;sN&quot;:5,&quot;aN&quot;:&quot;c3c1m1r1a1&quot;}"></button></div><div class="c-show-pipe x-hidden-vp-mobile-st"><a id="uhfCatLogo" class="c-logo c-cat-logo" href="https://www.microsoft.com/en-us/download/default.aspx" aria-label="Download Center" itemprop="url" data-m="{&quot;cN&quot;:&quot;CatNav_Download Center_nav&quot;,&quot;id&quot;:&quot;n6c3c1m1r1a1&quot;,&quot;sN&quot;:6,&quot;aN&quot;:&quot;c3c1m1r1a1&quot;}"><span>Download Center</span></a></div><div class="cat-logo-button-cont x-hidden"><button type="button" id="uhfCatLogoButton" class="c-cat-logo-button x-hidden" aria-expanded="false" aria-label="Download Center" data-m="{&quot;cN&quot;:&quot;Download Center_nonnav&quot;,&quot;id&quot;:&quot;nn7c3c1m1r1a1&quot;,&quot;sN&quot;:7,&quot;aN&quot;:&quot;c3c1m1r1a1&quot;}">
```
Comment (ShahafBenYakir) [inline Packs/FeedAzure/Integrations/FeedAzure/test_data/response_mock.txt:1]: Do we need all of this response data? can we somehow decrease the size of this file? Or it must be like this?

File: Packs/FeedAzure/Integrations/FeedAzure/FeedAzure_test.py (hunk 1 above)
Comment (teizenman) [reply Packs/FeedAzure/Integrations/FeedAzure/FeedAzure_test.py:206]: As stated in the Description - its duplicate with the PB, was written before the decision to have PB for the feed.

File: Packs/FeedAzure/Integrations/FeedAzure/test_data/response_mock.txt (hunk 2 above)
Comment (teizenman) [reply Packs/FeedAzure/Integrations/FeedAzure/test_data/response_mock.txt:1]: Its not entirely necessary, its just the actual response thats coming back from the api call.
Will truncate it.

**Pull request merged**